pkg/operator/sync.go

```diff
@@ -171,6 +172,28 @@ func (optr *Operator) syncMachineConfigController(config renderConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncEtcdQuorumGuard(config renderConfig) error {
+	eqgBytes, err := renderAsset(config, "manifests/etcdquorumguard/deployment.yaml")
```

Comment:
should this sync also wait for the deployment to correctly roll out? (see `waitForDeploymentRollout`)

Reply:
Yes.  Done.